tests/QAN/filters_test.js

```diff
@@ -294,9 +276,8 @@ Data(shortCutTests).Scenario(
 
 Scenario('PMM-T437 - Verify short-cut navigation for n/a items @qan', async ({ I, qanFilters }) => {
   qanFilters.waitForFiltersToLoad();
-  qanFilters.applyShowAllLink('Cluster');
-  qanFilters.checkLink('Cluster', 'ps-dev-cluster', true);
+  qanFilters.checkLink('Cluster', 'dev-cluster', true);
   I.fillField(qanFilters.fields.filterBy, 'n/a');
-  qanFilters.checkLink('Cluster', 'n/a', false);
-  qanFilters.checkLink('Replication Set', 'n/a', false);
+  qanFilters.checkLink('Cluster', 'undefined', false);
```

Comment:
undefined looks like a bug to me? @beatahandzelova

Reply:
No bug. It's still shown as "n/a" in UI, "undefined" is just used in element names.